main/inc/lib/database.constants.inc.php

```diff
@@ -172,6 +172,7 @@
 define('TABLE_STUDENT_PUBLICATION_REL_DOCUMENT', 'student_publication_rel_document');
 define('TABLE_STUDENT_PUBLICATION_REL_USER', 'student_publication_rel_user');
 define('TABLE_STUDENT_PUBLICATION_ASSIGNMENT_COMMENT', 'student_publication_comment');
+define('TABLE_COMPILATIO', 'plagiarism_compilatio_docs'); 
```

Comment:
Should be 
define('TABLE_PLAGIARISM', 'plagiarism_docs');
I don't think we need to differentiate the tool at this level (as you said, the table could be reused for other plagiarism tools later)